.env.example

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_SUBGRAPH_URL=https://api.thegraph.com/subgraphs/name/nounsdao/nouns-subgraph
```

Comment:
Not sure this need to live in the env as it should be the same for all/is public? maybe a constants? no strong opinion